resources/gen-ai/aws-contentgen-appsync-lambda/schema.graphql

```diff
@@ -0,0 +1,46 @@
+input ModelConfiguration {
+	provider: String
+	modelId: String
+	streaming: Boolean
+	model_kwargs: AWSJSON
+}
+
+
+input ImageInput {
+	jobid: ID
+	filename: String
+	model_config: ModelConfiguration
+	input_text: String!
+	negative_prompts: String
+}
+
+type ImageOutput @aws_iam @aws_cognito_user_pools {
```

Comment:
is there a possibility to generate multiple output images for one prompt/multiple prompts ?

Reply:
yes, amazon titan image generator support numberofimages property, it can generate maximum 5 pics with one prompt at one go. added the numberofimages in model_kwargs of the construct.